Ghidra/Features/Decompiler/ghidra_scripts/classrecovery/RTTIWindowsClassRecoverer.java

```diff
@@ -151,9 +151,8 @@ public List<RecoveredClass> createRecoveredClasses() throws Exception {
 		}
 		// otherwise figure everything out from scratch
 		else {
-			monitor.setMessage("Figuring out class method types");
+			Msg.debug(this, "Figuring out class method types");
```

Comment:
You clobbered the monitor. Did you intend to?

Reply:
yes,